sphinxcontrib/openapi/openapi30.py

```diff
@@ -262,6 +263,27 @@ def _httpresource(endpoint, method, properties, render_examples):
         for line in param.get('description', '').splitlines():
             yield '{indent}{indent}{line}'.format(**locals())
 
+    # print request content
+    if render_request:
+        request_content = properties.get('requestBody', {}).get('content', {})
+        if request_content:
+            schema = request_content['application/json']['schema']
```

Comment:
What if request's `content-type` is not `application/json`? It's not safe to assume it's alwats `applicaition/json`. Can we somehow handle this gracefully?

Reply:
I think, nobody who uses OpenAPI is interested in a content-type other than JSON.
What if I update the line #269 as follows:

if request_content and request_content['application/json']:

Reply:
That won't work. This is wrong assumption that nobody is interested, and I definitely do not want the extension to crash if there's a request body with non JSON content. For instance, at my work we're maintain an API endpoint that may receive data as CSV. In case I put `text/csv` to `requestBody` and omit `application/json`, the extension will crash. I'm not suggesting to *render* things other than JSON, yet I'm pretty much confident that we must not crash.

Reply:
In the current code, it won't crash. If other than JSON format is used it will silently continue without showing the request body as it works now.
It certainly can be updated by ones are interested in other formats.
BTW, this part looks more robust than proposed initially:

if request_content and 'application/json' in request_content: